modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFolderResourceImpl.java

```diff
@@ -156,32 +172,16 @@ public ObjectFolder putObjectFolder(
 			_objectFolderService.updateObjectFolder(
 				objectFolder.getExternalReferenceCode(), objectFolderId,
 				LocalizedMapUtil.getLocalizedMap(objectFolder.getLabel()),
-				transformToList(
```

Comment:
We will remove this parameter from Service and LocalService.
We will create a method to manage the related models like `_addObjectDefinitionResources()`.